extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -76,7 +76,7 @@ define('ext.wikia.adEngine.adContext', [
 			context.providers.openX = true;
 		}
 
-		context.slots.invisibleHighImpact = (
+		context.slots.invisibleHighImpact = qs.getVal('highimpactslot', false) || (
```

Comment:
Will it work with `0` and `1`? Probably `Querystring` will cast it to `false` and `true` but it's better to ask ;) And I'm asking since most of our URL params are described in the way they accept `0` or `1`.